packages/plugin-compat/sources/extensions.ts

```diff
@@ -290,4 +290,10 @@ export const packageExtensions: Array<[string, PackageExtensionData]> = [
       react: `^16.8.6`,
     },
   }],
+  // https://github.com/ant-design/react-slick/pull/95
+  [`@ant-design/react-slick@*`, {
```

Comment:
```suggestion
  [`@ant-design/react-slick@<=0.28.3`, {
```
Since it has been merged I'll lock the range to `<=0.28.3` which is the currently released version